<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1473</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>500</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="fit_params">
     <item row="9" column="1">
      <widget class="QCheckBox" name="param_enable_7">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QCheckBox" name="param_enable_10">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QCheckBox" name="param_enable_13">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Fitting Parameters</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QCheckBox" name="param_enable_11">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QDoubleSpinBox" name="param_0">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="param_label_0">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Scale Factor (a.u.)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QCheckBox" name="param_enable_8">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QCheckBox" name="param_enable_15">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QCheckBox" name="param_enable_12">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QCheckBox" name="param_enable_16">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QCheckBox" name="param_enable_14">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QCheckBox" name="param_enable_9">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QCheckBox" name="param_enable_17">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QLabel" name="param_label_11">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>SM, Width of M Distribution</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QLabel" name="param_label_10">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>M, Number of Layers</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="param_enable_0">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="15" column="2">
      <widget class="QLabel" name="param_label_13">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>del, Inter Plane Strain</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="QLabel" name="param_label_12">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>DAB, In plane strain</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="param_label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Background S^3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="param_label_5">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Background S^4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QLabel" name="param_label_8">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>d002, Interlayer Spacing (Angstrom)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QLabel" name="param_label_7">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>A, In Plane Cell Constant (Angstrom)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="param_label_6">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Background 1/S</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="18" column="2">
      <widget class="QLabel" name="param_label_16">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Debye Waller Temperature Factor, (Angstrom^2)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="param_label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Background S</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="param_enable_6">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="param_enable_1">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="param_enable_3">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="param_enable_2">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="param_enable_4">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="param_enable_5">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QDoubleSpinBox" name="param_14">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QDoubleSpinBox" name="param_13">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QDoubleSpinBox" name="param_11">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QDoubleSpinBox" name="param_12">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QDoubleSpinBox" name="param_15">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QDoubleSpinBox" name="param_16">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QDoubleSpinBox" name="param_17">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QDoubleSpinBox" name="param_1">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QLabel" name="param_label_9">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>La, Coherence Length</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QDoubleSpinBox" name="param_5">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QDoubleSpinBox" name="param_2">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="17" column="2">
      <widget class="QLabel" name="param_label_15">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Pt, Probability of 3R Stacking</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="19" column="2">
      <widget class="QLabel" name="param_label_17">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>PO, Preferential Orientation Factor</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="16" column="2">
      <widget class="QLabel" name="param_label_14">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Pr, Probability of Random Stacking</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QDoubleSpinBox" name="param_4">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QDoubleSpinBox" name="param_3">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="param_label_1">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Background Constant</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="param_label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Background S^2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QDoubleSpinBox" name="param_6">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QDoubleSpinBox" name="param_7">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QDoubleSpinBox" name="param_9">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QDoubleSpinBox" name="param_8">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QDoubleSpinBox" name="param_10">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="20" column="2">
      <widget class="QPushButton" name="load_scan">
       <property name="text">
        <string>Load XRD Pattern</string>
       </property>
      </widget>
     </item>
     <item row="20" column="0" colspan="2">
      <widget class="QPushButton" name="import_data">
       <property name="text">
        <string>Import Parameters</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="mplwindow" native="true">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>801</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="mplvl"/>
     </item>
    </layout>
   </widget>
   <zorder>param_label_2</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>label</zorder>
   <zorder>mplwindow</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1473</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
